security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/limitreduction/LimitReductionDeserializer.java

```diff
@@ -76,9 +75,17 @@ public LimitReduction deserialize(JsonParser parser, DeserializationContext dese
                 }
             }
         });
-        return new LimitReduction(context.limitType, context.value, context.monitoringOnly,
-                context.contingencyContext,
-                context.networkElementCriteria != null ? context.networkElementCriteria : Collections.emptyList(),
-                context.durationCriteria != null ? context.durationCriteria : Collections.emptyList());
+        LimitReduction.Builder builder = LimitReduction.builder(context.limitType, context.value)
+                .withMonitoringOnly(context.monitoringOnly);
+        if (context.contingencyContext != null) {
```

Comment:
What is the meaning of a null contingency context?

Reply:
We are in the deserializer here, the contingency context may not have been specified in the JSON file to import.

Reply:
Why do we have these two ways? Null and ALL ?

Reply:
Ok, I will set the ContingencyContext as mandatory in the JSON file.

Reply:
The ContingencyContext is now mandatory in the JSON files to import.